source/dpp/translation/type/package.d

```diff
@@ -434,11 +434,25 @@ private string translateUnexposed(in from!"clang".Type type,
     const translation =  translateString(spelling, context)
         // we might get template arguments here (e.g. `type-parameter-0-0`)
         .replace("type-parameter-0-", "type_parameter_0_")
+        .maybeTranslateTypeof()
```

Comment:
Why is translating it here the right place?

Reply:
This function (dpp.translation.type.translateUnexposed(), which is called from dpp.translation.type.translate()) is called from at least 2 places that I know of where typeof could appear:

- Translating the return type of a function (source/dpp/translation/function_.d, line 194)
- Translating the type of a variable (source/dpp/translation/variable.d, line 60; translateType is an alias for dpp.translation.type.translate())

I thought that it would make sense to translate it here, since both cases above call the same translate() function.